### DISCUSSION TITLE
Access permissions

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#17329] [How to]
Hello, I need to configure a profile with a user, I need this user to only be able to see and access the records assigned to him, for example contacts, accounts, etc...

How do I achieve this?

**Comment by Giorgos-Hellene** [comment]
Good morning ! You have to go Permissions -> Module access and define all modules as private. Also if you have other users on your instance you have to add custom sharing rules
